be/src/exec/vectorized/json_scanner.cpp

```diff
@@ -617,10 +617,13 @@ Status JsonReader::_read_and_parse_json() {
 #else
     // TODO: Remove the down_cast, should not rely on the specific implementation.
     StreamLoadPipeInputStream* stream_file = down_cast<StreamLoadPipeInputStream*>(_file->stream().get());
-    // For efficiency reasons, simdjson requires a string with a few bytes (simdjson::SIMDJSON_PADDING) at the end.
-    RETURN_IF_ERROR(stream_file->read_one_message(&_json_binary_ptr, &length, simdjson::SIMDJSON_PADDING));
-    if (length == 0) {
-        return Status::EndOfFile("EOF of reading file");
+    {
+        SCOPED_RAW_TIMER(&_counter->file_read_ns);
+        // For efficiency reasons, simdjson requires a string with a few bytes (simdjson::SIMDJSON_PADDING) at the end.
+        RETURN_IF_ERROR(stream_file->read_one_message(&_json_binary_ptr, &length, simdjson::SIMDJSON_PADDING));
```

Comment:
If the message arrives row by row, does it the counter will occupy much time?

Reply:
https://gist.github.com/stunpix/54c304bc010307562ec3434dbd5e76ae
With this code, I have done a rough benchmark about the `clock_gettime(MONOTONIC)`, which is the implement of `SCOPED_RAW_TIMER`. The result is about `20ns/call`. It seems fast enough.

Reply:
How many rounds do you run?